app-modules/ai/src/Services/Contracts/AiService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AdvisingApp\Ai\Services\Contracts;
+
+use AdvisingApp\Ai\Models\AiAssistant;
+use AdvisingApp\Ai\Models\AiThread;
+use AdvisingApp\Ai\Models\AiMessage;
+
+interface AiService
+{
+    /**
+     * This method is passed an unsaved `AiAssistant` model and should return
+     * the model with any additional data that associates it with
+     * the AI service, such as the `assistant_id`.
+     */
+    public function createAssistant(AiAssistant $assistant): void;
+
+    /**
+     * This method is passed an unsaved `AiAssistant` model and should update the
+     * AI service with the new details.
+     */
+    public function updateAssistant(AiAssistant $assistant): void;
```

Comment:
Same as above, does this return an `AiAssistant` after the update is done? If so the return type here should be updated.

Reply:
As above